pyglmnet/pyglmnet.py

```diff
@@ -1245,6 +1251,10 @@ def fit(self, X, y):
 
             glm.fit(X, y)
             glms.append(glm)
+
+        for glm in glms:
```

Comment:
we never return these `glm` objects to the user: they are internal to the `GLMCV` class. so do we even need to reset `_allow_refit` to `False`?

otherwise good to merge!

Reply:
Actually the best one is indeed available to the user: https://github.com/glm-tools/pyglmnet/blob/master/pyglmnet/pyglmnet.py#L1094